dataframe-csv/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/readTsv.kt

```diff
@@ -367,7 +369,8 @@ public fun DataFrame.Companion.readTsv(
  *
  * [DataFrame.readTsvStr][readTsvStr]`("a,b,c", delimiter = ",")`
  *
- * @param url The URL from which to fetch the data. Can also be compressed as `.gz` or `.zip`, see [Compression].
+ * @param url The URL from which to fetch the data.
```

Comment:
Could you please convert `@param url `to `@param [url]` ? it's just easier to read

Reply:
these are out of date generated sources. Not relevant to this pr